sw/host/rom_ext_image_signer/Cargo.toml

```diff
@@ -0,0 +1,25 @@
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+[package]
+name = "rom_ext_signer"
+version = "0.1.0"
+authors = ["lowRISC contributors"]
+edition = "2018"
+
+[profile.dev]
+panic = "abort"
+lto = false
+opt-level = "z"
+debug = true
```

Comment:
panic = abort isn't correct for host builds where we know we have an OS, and in dev we probably don't want opt-level = `z` at all on hosts (this is required on embedded to keep image size down).